MultiplexReservationSystem/src/main/java/p/lodz/pl/multiplexreservationsystem/repository/ScreeningsRepository.java

```diff
@@ -0,0 +1,22 @@
+package p.lodz.pl.multiplexreservationsystem.repository;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import p.lodz.pl.multiplexreservationsystem.model.Screenings;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Repository
+public interface ScreeningsRepository extends JpaRepository<Screenings, Long> {
+  @Query("Select s, m.title, m.duration From Screenings s left join Movies m on s.movieId = m.id")
+  List<Object[]> findAllScreeningsJoinMovies(Pageable page);
```

Comment:
Czemu Object[] a nie jakieś własne DTO?

Reply:
Umknęła mi ta pozostałość z początków aplikacji, gdzie miałem duży problem ze zrzutowaniem do DTO, w przypadku kiedy zapytanie wydobywa mi cały obiekt + 2 atrybuty z innego obiektu (tu widoczne w selekcie). Obecnie rzeczywiście można by było zrefaktoryzować to oraz ScreeningsDtoMappera do istniejącego już ScreeningsDto.